adapter/deploy/yaml/adapter.yaml

```diff
@@ -19,10 +19,12 @@ spec:
           args:
             - --with-stdout=true
           imagePullPolicy: Always
-          volumeMounts:
-            - mountPath: /etc/localtime
-              name: host-time
-              readOnly: true
+          env:
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                apiVersion: v1
+                fieldPath: metadata.namespace
```

Comment:
Why add namespace here?
What about timezone?

Reply:
This adapter is just a webhook that receives notifications and outputs. Most users don't need this. So I don't think it's necessary to include it in the helm.